.husky/_/husky.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
I think you can remove this husky.sh file. Why did you add this?
Reading the docs, I think it should be installed automatically in `.git/hooks`-folder after you `npx husky install`.

Reply:
This is the file that is added by installing husky. 

Husky created it. :-)

Reply:
Is this for Windows users then?
I think I installed husky in a clean project and it did not add this file 🤔